apps/mull-ui/src/app/pages/register/register.scss

```diff
@@ -0,0 +1,67 @@
+@import 'variables';
+
+.register-mull-logo {
+  padding-top: 5rem;
+  display: block;
+  margin: 0 auto;
+  width: 30%;
+  padding-bottom: 3rem;
+}
+
+.register-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-row-gap: 1.5rem;
+  margin: 0 auto;
+  width: 95%;
+}
+
+.register {
+  font-weight: bold;
+  color: #000000;
+  background-color: rgba(39, 176, 154, 0.6);
+  border: none;
+  height: 45px;
+  cursor: pointer;
+  border-radius: 16px;
+  font-size: 16px;
+  outline: none;
+}
+
+.custom-text-input-container,
+.register {
+  align-self: center;
+  justify-self: center;
+  width: 100%;
+}
+
+.login-here {
+  font-size: 14px;
+  text-decoration: none;
+  color: $primary-color;
+  font-weight: bold;
+}
+
+.login-link {
+  text-align: center;
+}
+
+@media (min-width: 600px) {
+  .register,
+  .custom-text-input-container {
+    width: 45%;
+  }
+  .register-mull-logo {
+    width: 15%;
+  }
+}
+
+@media (min-width: 1200px) {
+  .register,
+  .custom-text-input-container {
+    width: 22%;
+  }
+  .register-mull-logo {
+    width: 10%;
+  }
+}
```

Comment:
Same as for login, remove this and have the `.page-container-no-nav` suggestion.